pay-api/devops/gcp/clouddeploy.yaml

```diff
@@ -45,7 +45,8 @@ serialPipeline:
       container-name: "pay-api-test"
       cloudsql-instances: "gtksf3-test:northamerica-northeast1:pay-db-test"
       service-account: "sa-api@gtksf3-test.iam.gserviceaccount.com"
-      resources-memory: 1Gi
+      resources-cpu: 4000m
+      resources-memory: 8Gi
```

Comment:
@bolyachevets is this ok for test? Jia is doing some large statement testing

Reply:
She's going to try 2GI first

Reply:
sure. i set to prod values, it will get overwritten to the values in clouddeploy.yaml next time you redeploy